imports/api/accounts/methods.js

```diff
@@ -131,6 +131,41 @@ export const insertAccountOnSignUp = new ValidatedMethod({
 });
 
 
+export const updateProfile = new ValidatedMethod({
+    name: 'updateProfile',
+    mixins: [LoggedInMixin],
+    checkLoggedInError: {
+        error: 'notLogged',
+        message: 'You need to be logged in to update profile'
+    },
+    validate: new SimpleSchema({
+        'users.name': {
```

Comment:
make another validation `users` type `Object` in order to follow standard way of this application.